src/modules/home/HomePage.js

```diff
@@ -15,15 +16,21 @@ import FontAwesome from 'react-fontawesome';
 import './index.css';
 import Config from '../../config';
 import { hashHistory } from 'react-router';
-import appSession$ from './../../modules/app/appSession';
 import Logos from '../../components/logos/Logos';
 
-const reportURL = Config.magasin.urls.client.report.goToReport(appSession$);
-const magasinURL = Config.magasin.urls.client.magasin.goToMagasin(appSession$);
-const analysisURL = Config.magasin.urls.client.analysis.addAnalysis(appSession$);
+const reportURL = Config.magasin.urls.client.report.goToReport;
+const magasinURL = Config.magasin.urls.client.magasin.goToMagasin;
+const analysisURL = Config.magasin.urls.client.analysis.addAnalysis;
 const aboutURL = '/about';
 const notFoundURL = '/notfound';
+import inject from 'react-rxjs/dist/RxInject';
```

Comment:
Move this importeres above the constants. So that the imports will collapse automatically when opening the file in idea.